src/pykiso/lib/auxiliaries/udsaux/uds_auxiliary.py

```diff
@@ -250,3 +283,38 @@ def _abort_command(self) -> None:
     def _run_command(self, cmd_message, cmd_data=None) -> Union[dict, bytes, bool]:
         """Not used."""
         pass
+
+    def _get_instance(self):
+        return self
+
+
+class TesterPresentSender(threading.Thread):
+    """Self that continuously sends tester present messages via UDS"""
+
+    TESTER_PRESENT_SEND = b"\x3E\x00"
+
+    def __init__(self, uds_aux: UdsAuxiliary, period: float = 4):
```

Comment:
to be honest you can define a range for the period having the tester present send every 10 ms doesn't make sense

Reply:
@BKaDamien 
How would it look like with a range ? and why do you assume every 10 ms ? isnt it 4 here ?

Reply:
I think what Damien means is that it doesn't make sense to let the possibility to define a period like 10ms (and use a lot of resources) when the goal is only to send a message at least every 5 seconds